### DISCUSSION TITLE
Infiniband

### DISCUSSION THREAD

**Discussion by prodchia** [Chia-Network/bladebit#81] [Q&A] [answered]
I have two servers each with 256GB RAM and 20 cores E52673-V4. Can I use bladebt If i connect the two servers with a 40Gbs infiniband connection?

**Comment by harold-b** [comment, marked as answer]
No, Bladebit is not design for such scenario. Even if you were able to virtualize the connection as memory, the performance would be appalling. 

5 GB/s (40Gb/s) is in the range of SSD speeds. Compare that with something like a quad channel DDR4 @ 2400MHz which would give you 76.8 GB/s, not to mention something like 8 channel @ 3200MHz which would be in the 200 GB/s range.

In such a scenario SSD plotting is better.
